### ISSUE TITLE
health check UI brainstorming

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/ecosystem#138]
Here's an alternative UI for the health check. This displays the information proportional to the changes in the PR - lists the changes files and affected packages. It mixes the different checks into one format (so has the drawback that it's more work to make some of the checks optional).

cc @mosuem 

---

__Changed Files__

| File | Notes | Coverage |
| --- | --- | --- |
| ✔️ pkgs/foo/lib/main.dart |  | 💚 100 % |
| ⚠️ pkgs/foo/bin/main.dart | License header missing; ... | 💔 Not covered |
| ✔️ pkgs/foo/test/main.dart |  | 💚 40 % ⬆️ 1 % |

__Packages__

No publishable packages affected by this PR.

-or-

| Package | Changes | Changelog |
| --- | --- | --- |
| ⚠️ package:foo (pkgs/foo) | 4 changed files | No changelog update <br> Please provide ... |
| ✔️ package:bar (pkgs/bar) | 2 changed files | - Lorem ipsum 1 ...<br>- Lorem ipsum 1 ... |

**Comment by devoncarew** [comment]
@mosuem - any thoughts here? These ideas may be a bit out of date now, but I think it would be worth taking another pass at the current health check output given that we'll probably be enabling it on more of our repos.

**Comment by mosuem** [comment]
Yeah happy about any ideas for a nicer UX! We also probably want to link to more information about everything and add more docs while we are at it...
